src/Datadog.Trace/Conventions/OtelOutboundHttpConvention.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tagging;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Conventions
+{
+    internal class OtelOutboundHttpConvention : IOutboundHttpConvention
+    {
+        private static HttpTagsKeysMapping tagKeys = new HttpTagsKeysMapping
+        {
+            Method = Trace.Tags.HttpMethod,
+            Url = Trace.Tags.HttpUrl,
+            StatusCode = Trace.Tags.HttpStatusCode,
+            HandlerType = "http-client-handler-type",
+            InstrumentationName = Trace.Tags.InstrumentationName,
+        };
+
+        private readonly Tracer _tracer;
+
+        public OtelOutboundHttpConvention(Tracer tracer)
+        {
+            _tracer = tracer;
```

Comment:
As you commented earlier, this reference is kind of ugly. The conventions are meant to differentiate spans right? Perhaps, we can have a Tracer create the scope first (not inside this method) and this method is only responsible for setting the tags or modifying the span, but that can all be done without the Tracer.

Reply:
I did the way you suggest initially, but modifying the span's `OperationName` and setting the `HttpTag`'s tag mapping after the object is created looked even worse to my eyes.

This is the problematic code:
```
tags = new HttpTags(tagKeys);  // sets the tags mapping conventions
string serviceName = _tracer.Settings.GetServiceName(_tracer, "http-client"); // I do not know what it is yet :) 
var scope = _tracer.StartActiveWithTags("http.request", tags: tags, serviceName: serviceName, spanId: args.SpanId); // this sets some stuff 
```

I am not sure yet if we want to have a different ServiceName, OperationName for OTel and Datadog.

Maybe I will be able to refactor it if I simply know more about OTel and be more familiar with this repo 😉

Reply:
No I don't think this is a blocker. Maybe later we can revisit this to make it cleaner, but anyways we will need to make a runtime decision based on how the Tracer is configured, and this accomplishes that 👍🏼

Reply:
I'm missing something for sure, but, why not use `Tracer.Instance` instead? If the configuration is changed at runtime we cannot pick the new configuration. Also removes this .ctor.

Reply:
Good thanks for clarifying.